integrasjon/dokgen-klient/src/main/java/no/nav/foreldrepenger/melding/integrasjon/dokgen/dto/innvilgelsefp/ForeldrepengerInnvilgelseDokumentdata.java

```diff
@@ -219,6 +220,10 @@ public int getPrematurDager() {
         return prematurDager;
     }
 
+    public int getAntallDødeBarn() { return antallDødeBarn; }
+
+    public String getDødsdato() {return dødsdato; }
+
```

Comment:
Ikke skriv gettere manuelt og på en linje, men bruk ALT+Insert i IntelliJ for å autogenerere. IntelliJ collapser dem automatisk til en linje i visningen, spacer blir riktig, og formatet blir konsistent med de øvrige getterne i klassen.